### PR TITLE
add seqsero2 module

### PR DESCRIPTION
**Pull request by rpetit3**
## PR checklist

Closes https://github.com/nf-core/modules/issues/943

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: modules/seqsero2/main.nf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    tuple val(meta), path("results/*_result.tsv"), emit: tsv
+    tuple val(meta), path("results/*_result.txt"), emit: txt
+    path "versions.yml"                          , emit: versions
+
+    script:
+    def prefix = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    """
+    SeqSero2_package.py \\
+        $options.args \\
+        -d results/ \\
```
Comment (sguizard) [inline modules/seqsero2/main.nf:35]:
```suggestion
        -d ./ \\
```
It's preferred to don't use custom name output folder. Would it work like this?

**Review by sguizard — COMMENTED** [review]
Just one comment, otherwise looks good to me.

File: modules/seqsero2/main.nf (hunk 1 above)
Comment (rpetit3) [reply modules/seqsero2/main.nf:35]: Let's give it a shot here! Tried locally and it didn't like ./ (Had to do with symbolic link of input, which I think is seqsero2 related)

**Comment by rpetit3** [comment]
@sguizard here's the error when using `./`

```
Error: ELOOP: too many symbolic links encountered, stat '/home/runner/pytest_workflow_9vdgx15q/seqsero2_test_seqsero2/work/bb/bee883e15aff2d44afcf2dc0351f6d/genome.fasta'
```

**Review by sguizard — APPROVED** [review]
OK for me 👍

**Pull request merged**